src/index.js

```diff
@@ -2,13 +2,30 @@ import express from 'express';
 import bodyParser from 'body-parser';
 import cors from 'cors';
 import passport from 'passport';
+import methodOverride from 'method-override';
+import morgan from 'morgan';
 import errorhandler from 'errorhandler';
+import cookieSession from 'cookie-session';
 import router from './routes/index';
 
 const isProduction = process.env.NODE_ENV === 'production';
 
 // Create global app object
 const app = express();
```

Comment:
Would you please get rid of this comment

Reply:
@hakoemmy  some comments were there even before 
so I don't think i am allowed to remove them 

for example, the one you're suggesting me to remove

Reply:
Ok. @blaise82 Remove yours only